frontend/packages/knative-plugin/src/components/add/event-sources/EventSourcesSelector.tsx

```diff
@@ -35,6 +35,19 @@ const EventSourcesSelector: React.FC<EventSourcesSelectorProps> = ({ eventSource
     },
     [setFieldValue, setFieldTouched, validateForm],
   );
+
+  if (eventSourceItems === 1) {
+    return (
+      <ItemSelectorField
+        itemList={eventSourceList}
+        loadingItems={!eventSourceItems}
+        name="type"
+        onSelect={handleItemChange}
+        autoSelect
+      />
+    );
+  }
+
```

Comment:
Instead of doing this, you can add this check ` (eventSourceItems === 1) ` where you are adding `title="Type"` in FormSection. You can pass <empty-string> to title if eventSourceItems is 1, else pass `Type`.

Reply:
I agree with @debsmita1 
```
const itemSelector =  <ItemSelectorField
        itemList={eventSourceList}
        loadingItems={!eventSourceItems}
        name="type"
        onSelect={handleItemChange}
        autoSelect
      />

return eventsourceItem > 1 ? <FormSection title="Type">{itemSelector}</FormSection> : itemSelector;
```

Reply:
passing an empty string to title results in rendering an empty `<div>` for form section causing extra spacing.
![extra-space-event-form](https://user-images.githubusercontent.com/38663217/80377196-939b2600-88b8-11ea-8933-b0b8903f67d2.png)

Reply:
Look at the code I pasted.. it shouldn't result in any spacing.

Reply:
updated code to use `?` operator instead of `if` block